charts/sauron/newman/README.md

```diff
@@ -15,6 +15,10 @@ Newman Tests run against production and goes from Andi Org creation through Disc
 
 ## To Run the suite
 - Enter Sauron (in environment or locally)
+  - Find the name of a previously run sauron cron pod - i.e:
+```shell
+oc debug sauron-cron-27900780--1-847ds
+```
```

Comment:
Will this execute the sauron deploy script as part of spinning up the pod? I suppose that's fine if so.

Reply:
It does not execute the entrypoint command